legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-dbExtension/legend-engine-xt-relationalStore-memsql/legend-engine-xt-relationalStore-memsql-pure/src/main/resources/core_relational_memsql.definition.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "core_relational_memsql",
-  "pattern": "(meta::relational::functions::sqlQueryToString::memsql|meta::relational::tests::sqlQueryToString::memsql|meta::relational::memsql::tests)(::.*)?",
+  "pattern": "(meta::relational::functions::sqlQueryToString::memsql|meta::pure::alloy::connections|meta::protocols::pure|meta::relational::tests::sqlQueryToString::memsql|meta::relational::memsql::tests)(::.*)?",
```

Comment:
There should be no reference to Alloy in here. Lets look to refactor whatever needs this package